### DISCUSSION TITLE
Task Scheduler - On shutdown instead of boot-up?

### DISCUSSION THREAD

**Discussion by Greg-ATG** [007revad/Synology_HDD_db#323] [General]
According to "how_to_schedule.md", the task should be run on boot-up.  Is there a specific reason for this?  Some changes that the script makes require a subsequent reboot, so wouldn't it kill two birds with one stone to run the script during shutdown?

**Comment by 007revad** [comment]
Some people actually schedule it to run at shut down and at boot. 

You should only need to reboot if the script has enable an unsupported PCIe card, or if m2 volumes were enabled on a model that does not support M2 volumes (and an NVMe drive installed).

Some Xpenology repos use my syn_hdd_db script as an addon, and they set it to run as a service during boot, to avoid having to reboot.
